examples/backend/endpoint.ml

```diff
@@ -14,6 +14,14 @@ module Counter_vdom = struct
   let about () = get (~/"counter-vdom" / "about")
```

Comment:
It's a detail, I grant you, but as `endpoints` are also used in the client, I suggest adding a `shared` library as a dependency on `backend` and `frontend` in which you expose the endpoints?